extensions/amp-form/amp-form.md

```diff
@@ -60,28 +60,34 @@ Example:
         </label>
         <input type="submit" value="Subscribe">
     </fieldset>
+    <div submit-success>
+        Subscription successful!
+    </div>
+    <div submit-error>
+        Subscription failed!
+    </div>
 </form>
 ```
 
 ## Attributes
 
 **target**
-__required__
+__required__ for `GET` requests.
```

Comment:
[requiring `target` doesn't make sense for XHR requests](https://github.com/ampproject/amphtml/issues/5607#issuecomment-253894602) IMO.